src/engine/controls/loopingcontrol.h

```diff
@@ -76,6 +76,17 @@ class LoopingControl : public EngineControl {
         mixxx::audio::FramePos startPosition = mixxx::audio::kInvalidFramePos;
         mixxx::audio::FramePos endPosition = mixxx::audio::kInvalidFramePos;
         LoopSeekMode seekMode = LoopSeekMode::None;
+
+        LoopInfo(mixxx::audio::FramePos start,
+                mixxx::audio::FramePos end,
+                LoopSeekMode mode) noexcept
+                : startPosition(start),
+                  endPosition(end),
+                  seekMode(mode) {
+        }
+
+        LoopInfo() noexcept {
+        }
```

Comment:
this makes LoopInfo not an aggregate anymore. This makes it less convenient to use. Please remove.